README.md

```diff
@@ -194,3 +194,10 @@ docker run --name gridproxy -e MNEMONICS="" -e SUBSTRATE="wss://tfchain.dev.grid
   ```bash
   helm install gridproxy/gridproxy
   ```
+
+# Release
```

Comment:
Should be ## Release instead of Release, that section should be moved into another file e.g release.md

Reply:
another pr planning to add a separate file for this part [here](https://github.com/threefoldtech/tfgridclient_proxy/pull/258). just wanted to write a quick note.
ok, will make the heading smaller.